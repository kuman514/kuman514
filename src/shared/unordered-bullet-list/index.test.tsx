import { render } from '@testing-library/react';

import UnorderedBulletList from '.';

describe('UnorderedBulletList', () => {
  it('should show as expected', () => {
    const { container } = render(
      <UnorderedBulletList>
        <li>Yasuo</li>
        <li>Koishi</li>
        <li>Hoshino</li>
      </UnorderedBulletList>
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should show as expected 2', () => {
    const { container } = render(
      <UnorderedBulletList>
        <li>
          <a href="yasuo-science.com">Yasuo</a>
        </li>
        <li>
          <a href="koishi.com">Koishi</a>
        </li>
        <li>
          <a href="hoshino.com">Hoshino</a>
        </li>
      </UnorderedBulletList>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
