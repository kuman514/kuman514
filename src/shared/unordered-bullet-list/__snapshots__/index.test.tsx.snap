// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnorderedBulletList should show as expected 1`] = `
<ul
  class="list-disc pl-8"
>
  <li>
    Yasuo
  </li>
  <li>
    Koishi
  </li>
  <li>
    Hoshino
  </li>
</ul>
`;

exports[`UnorderedBulletList should show as expected 2 1`] = `
<ul
  class="list-disc pl-8"
>
  <li>
    <a
      href="yasuo-science.com"
    >
      Yasuo
    </a>
  </li>
  <li>
    <a
      href="koishi.com"
    >
      Koishi
    </a>
  </li>
  <li>
    <a
      href="hoshino.com"
    >
      Hoshino
    </a>
  </li>
</ul>
`;
